custom_components/jlrincontrol/__init__.py

```diff
@@ -78,7 +78,7 @@
 CONF_PRESSURE_UNIT = "pressure_unit"
 CONF_HEALTH_UPDATE_INTERVAL = "health_update_interval"
 
-PLATFORMS = ["sensor", "lock", "device_tracker"]
+PLATFORMS = ["sensor", "lock", "switch, "device_tracker"]
```

Comment:
forgot a quote. should be PLATFORMS = ["sensor", "lock", "switch", "device_tracker"]

Reply:
have you ever test your code before PR? I've found an error when getting roc target value.